js_modules/dagit/packages/core/src/runs/RunUtils.tsx

```diff
@@ -102,6 +102,8 @@ export function handleLaunchResult(
       });
     }
     document.dispatchEvent(new CustomEvent('run-launched'));
+  } else if (obj.__typename === 'InvalidSubsetError') {
+    showCustomAlert({body: obj.message});
   } else if (obj.__typename === 'PythonError') {
     showCustomAlert({
```

Comment:
```suggestion
  } else if (obj.__typename === 'InvalidSubsetError || obj.__typename === 'PythonError') {
    showCustomAlert({
```

Reply:
Actually, doesn't the existing `else` clause at the bottom handle this correctly already?